### PR TITLE
DDF add manufacturer name "frient A/S" for keypad KEPZB-110

### PR DESCRIPTION
**Pull request by Idaho947**
If frient is not mention it doesn't work why this keypad.

### CODE REVIEW

**Review by Idaho947 — COMMENTED** [review]
As you ask I hope in order to add two manufactername and modelid

**Comment by Smanar** [comment]
The file edition haven't worked ^^.
But yes if you write 

```
"manufacturername": ["Develco Products A/S", "frient A/S"],
"modelid": ["KEPZB-110", "KEPZB-110"],
```

The DDF will work for both device "Develco Products A/S"-"KEPZB-110" AND "frient A/S"-"KEPZB-110"

File: devices/frient/kepzb-110_keypad.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
    "schema":"devcap1.schema.json",
-   "manufacturername":"Develco Products A/S",
```
Comment (Idaho947) [inline devices/frient/kepzb-110_keypad.json:3]: "manufacturername": ["Develco Products A/S", "frient A/S"],

File: devices/frient/kepzb-110_keypad.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+   "manufacturername":"frient A/S",
    "modelid":"KEPZB-110",
```
Comment (Idaho947) [inline devices/frient/kepzb-110_keypad.json:4]: "modelid": ["KEPZB-110", "KEPZB-110"],

**Comment by Smanar** [comment]
Easier.
Just edit (again) the file on your fork https://github.com/Idaho947/deconz-rest-plugin/tree/patch-1/devices/frient
The PR will be updated itself.

**Comment by Idaho947** [comment]
Ok done thanks

**Comment by manup** [comment]
Info: The PR is merged in https://github.com/dresden-elektronik/deconz-rest-plugin/commit/dacfc6fac26c5b81518a42d15a554ec93ee9180f, but somehow GitHub doesn't show it as such.

**Pull request merged**